homeassistant/components/hassio/__init__.py

```diff
@@ -124,6 +126,33 @@
 }
 
 
+async def async_addon_is_installed(
```

Comment:
Add `@bind_hass`

Reply:
I think that we should change this API to something like

`async_get_addon_info(…)`.

It will include all known information about the add-on, including if it is installed.

Reply:
Yeah, I'm using that for the next helper that was checking if the addon is started.

Ok.